impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -35,6 +35,415 @@ define(["./_focus"], function(focusUtil) {
     delete: 46
   };
 
+  function makeAccessibleSelect(elem) {
```

Comment:
Maybe the name should be `makeAccessibleListbox`. Isn't that the ARIA/WCAG pattern being implemented? https://www.w3.org/WAI/ARIA/apg/patterns/listbox/